components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs

```diff
@@ -1087,7 +1087,10 @@ IJsonSerializer jsonSerializer
                     OpcAuthenticationUsername = model.OpcAuthenticationUsername,
                     DataSetWriterGroup = model.DataSetWriterGroup,
                     DataSetWriterId = model.DataSetWriterId,
-                    DataSetPublishingInterval = model.DataSetPublishingInterval,
+                    DataSetPublishingIntervalTimespan = model.DataSetPublishingIntervalTimespan,
+                    DataSetPublishingInterval = model.DataSetPublishingIntervalTimespan.HasValue
```

Comment:
Shouldn't this if/else be the other way around ? Now if `DataSetPublishingIntervalTimespan` has a value, then `DataSetPublishingInterval` would also be assigned a value. And if the first one does not have a value, then value of `model.DataSetPublishingInterval` would be ignored.

Reply:
right ...